target/imagebuilder/files/Makefile

```diff
@@ -206,5 +221,18 @@ endif
 		$(if $(PACKAGES),USER_PACKAGES="$(PACKAGES)") \
 		$(if $(BIN_DIR),BIN_DIR="$(BIN_DIR)"))
 
-.SILENT: help info image
+list:
+ifneq ($(PROFILE),)
```

Comment:
Instead of creating this twice there could be a function called `_check_profile` used both by `image` and `list`